### DISCUSSION TITLE
Livewire resets playing animations

### DISCUSSION THREAD

**Discussion by Lexian-droid** [livewire/livewire#8740] [1. Help]
Hello, so I am making a website that uses Livewire, specifically so that I can play music across pages, but I have a problem, an animation that plays during music playback will reset everytime the page reloads, I've tried wire:ignore, the css is in the head, the navbar has wire:navigate, but the animation still resets. Any way to fix it? I am willing to try to use alpine, if someone could show me what I would need to do.

I've made a codepen, so that you can see my usage of the code https://codepen.io/Lexian-droid/pen/yLdzLGK

Also, this is what happens:

https://github.com/user-attachments/assets/18266bb0-b387-4903-87d5-1b34466ac487

**Comment by danie-ramdhani** [comment]
Have you tried `@persist`? https://livewire.laravel.com/docs/navigate#persisting-elements-across-page-visits

**Reply by Lexian-droid** [reply]
Yes, I have tried persist as well, but unfortunately the animation still resets.
